packages/create-webstone-app/src/helpers.ts

```diff
@@ -25,6 +41,12 @@ export const displayWelcome = () =>
   });
 
 export const displayNextSteps = async (ctx: Ctx) => {
+  const wsDevCommandPerPackageManager = {
+    npm: "npx ws dev",
```

Comment:
```suggestion
    npm: "npm ws dev",
```

Reply:
If `ws` isn't installed locally, yes. From their docs:

> Executes <command> either from a local node_modules/.bin, or from a central cache, installing any packages needed in order for <command> to run.

Unfortunately, `npm ws dev` doesn't work which is why we need this rather ugly key/value map here :(. I definitely preferred a solution without `npx`, but don't know of a way to make it work.